cranelift/codegen/src/isa/x64/inst.isle

```diff
@@ -1830,47 +1846,52 @@
                  src1
                  src2))
 
+;; Helper for emitting immediates with an `i64` value. Note that
+;; integer constants in ISLE are always parsed as `i64`s; this enables
```

Comment:
`i128`s, actually! (This may be an outdated comment from somewhere else?)

Reply:
Yep, it used to be in the middle of the `imm` rules, so I moved it above. I'll fix the comment :+1: